README.md

```diff
@@ -822,6 +823,30 @@ statusProvider:
       - Travis CI - Branch # or whatever builds and pushes your source image
 ```
 
+### Ruby Gems Index (`gem`)
+
+Pushes a gem [Ruby Gems](https://rubygems.org).
+It also requires you to be logged in `gem login`.
```

Comment:
```suggestion
It also requires you to be logged in with `gem login`.
```

Reply:
For the future, Is there an option to provide a RubyGems API token to `gem` command, so we don't have to go through `gem login` process?

Reply:
I think you can export an env variable but we need to check that.